tests/autoscaling/test_autoscaling_service_lib.py

```diff
@@ -320,10 +320,10 @@ def test_mesos_cpu_metrics_provider_filter_bogus_values_big_cpu_limit():
     |  inst  | prev cputime | elasped_time | cputime | limit |  norm_cputime  |      utilization      |
     +--------+--------------+--------------+---------+-------+----------------+-----------------------+
     | 1      |          126 |          600 |     480 |   2.1 | 480/(2.1-0.1)  | (240-126)/600 => 0.19 |
-    | 2      |            0 |          600 |    9600 |   2.1 | 9600/(2.1-0.1) | 4800/600 => 8.0       |
+    | 2      |            0 |          600 |    2400 |   2.1 | 2400/(2.1-0.1) | 1200/600 => 2.0       |
     | 3(bug) |            0 |          600 |  987654 |   2.1 | 9600/(2.1-0.1) | 493827/600 => 823     |
     | -      |            - |            - |       - |     - | -              | -                     |
-    | avg    |              |              |         |       |                | 4.095                 |
+    | avg    |              |              |         |       |                | 1.095                 |
```

Comment:
I'm not sure I understand why this (and the following tests) needed to change -- afaict all this logic should still work exactly as it did before.

Reply:
There are checks to make sure boxes don't use more cpus than the limit (cpus + burst) allowed. Essentially, the tests we do explicitly use numbers that were good for multiplicative burst, but too high for additive burst. As such, if I left the tests as they were, they would break because the tests would see that more cpus are being used than the limit allows. So, I changed the tests to maintain their intent - testing the limit check.

Reply:
Ah, gotcha.